crates/hyperdrive-math/src/long/open.rs

```diff
@@ -107,50 +107,50 @@ impl State {
 
     /// Calculate an updated pool state after opening a long.
     ///
-    /// For a given base amount and bond amount, base is converted to
+    /// For a given base delta and bond delta, the base delta is converted to
     /// shares and the reserves are updated such that
-    /// `state.bond_reserves -= bond_amount` and
-    /// `state.share_reserves += base_amount / vault_share_price`.
+    /// `state.bond_reserves -= bond_delta` and
+    /// `state.share_reserves += base_delta / vault_share_price`.
     pub fn calculate_pool_state_after_open_long(
         &self,
         base_amount: FixedPoint,
-        maybe_bond_amount: Option<FixedPoint>,
+        maybe_bond_deltas: Option<FixedPoint>,
```

Comment:
Drop the `s` on singular arg.
Since they are both deltas, we use consistent names.

Also, I think we can start dropping the `maybe_` prefix on `Option` types. The name doesn't need to carry the burden of informing users that it's an `Option` type and after performing a few GH searches it looks like it's more common to just name the argument as normal.

```diff
-        base_amount: FixedPoint,
-        maybe_bond_deltas: Option<FixedPoint>,
+        base_delta: FixedPoint,
+        bond_delta: Option<FixedPoint>,
```

Reply:
The maybe was something Alex implemented and expressed a preference for when I suggested we drop it some time ago. It seems to be standard practice in Rust. I originally also thought it was unnecessary, but it does seem to be the standard practice in stdlib so we should probably keep it. Either way, you'll want to take it up with alex.

Reply:
Cool, I'll acquiesce for now 🤷‍♂️